### PR TITLE
update auth0 related configuration

### PR DESCRIPTION
**Pull request by imcaizheng**
Replace auth0 audience configuration
```
export AUTH0_AUDIENCE=https://m2m.topcoder-dev.com/
export AUTH0_AUDIENCE_FOR_BUS_API=https://m2m.topcoder-dev.com/
```
with
```
export AUTH0_AUDIENCE=https://m2m.topcoder-dev.com/
export AUTH0_AUDIENCE_UBAHN=https://u-bahn.topcoder.com
```

Source code and documentation are updated to reflect the change.

### CODE REVIEW

File: src/common/helper.js [hunk 1]
```diff
@@ -651,8 +649,8 @@ module.exports = {
     }
     return ensureUbhanUserId({ userId })
   },
-  getM2Mtoken,
-  getTopcoderM2MToken,
+  getM2MToken,
```
Comment (imcaizheng) [inline src/common/helper.js:652]: Export  functions `getM2MToken` and `getM2MUbahnToken` though the functions are currently being used merely inside `helper.js`

File: src/common/helper.js [hunk 2]
```diff
@@ -394,7 +392,7 @@ async function getUserById (userId, enrich) {
  * @returns the request result
  */
 async function createUbhanUser ({ handle, firstName, lastName }) {
```
Comment (maxceem) [inline src/common/helper.js:394]: @imcaizheng here is a typo `createUbhanUser` => `createUbahnUser`

Comment (maxceem) [reply src/common/helper.js:394]: never mind, this is existent code

Comment (imcaizheng) [reply src/common/helper.js]: @maxceem Fixed.

**Review by maxceem — APPROVED** [review]
@imcaizheng works good as per local testing.

**Pull request merged**